src/components/Main/Main.tsx

```diff
@@ -9,7 +9,7 @@ import { Col, Row } from 'reactstrap'
 
 import { AllParams, EmpiricalData, Severity } from '../../algorithms/types/Param.types'
 import { AlgorithmResult } from '../../algorithms/types/Result.types'
-import { run } from '../../algorithms/run'
+import { run } from '../../workers/algorithm'
```

Comment:
Aim to keep the 'run' import signature the same as before the webworker implementation.